static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -182,7 +182,7 @@ define([
         // Get this browser's mvt test id.
             mvtCookieId = mvtCookie.getMvtValue();
 
-        if (smallestTestId <= mvtCookieId && largestTestId > mvtCookieId) {
```

Comment:
interestingly, our range for client side ab tests is 899999 and our range for server side ones is 100000, and we have one id left over that would never get in a test: https://github.com/guardian/frontend/blob/77e10c3bfd40fc4910546796d8aecc1ab7739f5e/static/src/javascripts/projects/common/modules/analytics/mvt-cookie.js#L11